seirmo/models.py

```diff
@@ -82,4 +79,10 @@ def simulate(self, parameters, times):
         sol = solve_ivp(lambda t, y: self._right_hand_side(t, y, c),
                         [times[0], times[-1]], y_init, t_eval=times)
 
-        return sol['y'].transpose()
+        if return_incidence is False:
```

Comment:
an easier construction (and maybe also easier to read) is
```
output = sol['y'].transpose()

if not return_incidence:
   return output

# Total infected is infectious `i` plus recovered `r`
total_infected = output[2, :] + output[3, :]

# Number of incidences is the increase in total_infected between the time points
n_incidence = total_infected[1:] - total_infected[:-1]

# Append n_incidence to output
output = np.vstack(tup=(output, n_incidence))

return output
```

Reply:
I think the total_infected should be output[:, 2] + output[:, 3] because we have transposed the solution matrix? I am also wondering, the length of n_incidence is one less than our output, could we stack them successful?